### PR TITLE
Added the main page wireframe

### PR DESCRIPTION
**Pull request by Riya5915**
Added landing page

### CODE REVIEW

File: public/landing-page/landing-page.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <!-- bootstrap stylesheet -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    
+    <!-- bootstrap scripts -->
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```
Comment (Omkaragrawal) [inline public/landing-page/landing-page.html:11]: move it to the bottom inside the body tag.

File: public/landing-page/landing-page.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>
```
Comment (Omkaragrawal) [inline public/landing-page/landing-page.html:12]: move it to the bottom inside the body tag.

File: public/landing-page/landing-page.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,900;1,100;1,700;1,900&family=Fira+Sans:ital,wght@1,300&family=IBM+Plex+Serif:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Satisfy&family=Yeseva+One&family=Montserrat:ital,wght@0,300;0,400;0,900;1,300;1,800;1,900&family=Ubuntu&display=swap" rel="stylesheet"> 
+    
+    <!-- font awesome for icons -->
+    <script defer src="https://use.fontawesome.com/releases/v5.0.7/js/all.js"></script>
```
Comment (Omkaragrawal) [inline public/landing-page/landing-page.html:19]: move it to the bottom inside the body tag.

**Review by Omkaragrawal — CHANGES_REQUESTED** [review]

File: public/landing-page/landing-page.html (hunk 1 above)
Comment (Riya5915) [reply public/landing-page/landing-page.html]: updated the changes.

File: public/landing-page/landing-page.html (hunk 2 above)
Comment (Riya5915) [reply public/landing-page/landing-page.html]: updated the changes

**Comment by Riya5915** [comment]
Updated landing-page

**Pull request merged**